subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -67,6 +68,34 @@ BUILD_ASSERT(!HAL_RADIO_GPIO_LNA_OFFSET_MISSING,
 	     HAL_RADIO_GPIO_LNA_OFFSET_PROP_NAME);
 #endif	/* FEM_NODE */
 
+#if defined(HAL_RADIO_GPIO_HAVE_PA_PIN) || defined(HAL_RADIO_GPIO_HAVE_LNA_PIN)
+static const nrfx_gpiote_t gpiote_palna = NRFX_GPIOTE_INSTANCE(
+	NRF_DT_GPIOTE_INST(FEM_NODE, HAL_RADIO_GPIO_PA_PROP));
+static uint8_t gpiote_ch_palna;
+
+BUILD_ASSERT(NRF_DT_GPIOTE_INST(FEM_NODE, HAL_RADIO_GPIO_PA_PROP) ==
+	     NRF_DT_GPIOTE_INST(FEM_NODE, HAL_RADIO_GPIO_LNA_PROP),
+	HAL_RADIO_GPIO_PA_PROP_NAME " and " HAL_RADIO_GPIO_LNA_PROP_NAME
+	" GPIOs must use the same GPIOTE instance.");
+#endif
+
+#if defined(HAL_RADIO_FEM_IS_NRF21540)
+static const nrfx_gpiote_t gpiote_pdn = NRFX_GPIOTE_INSTANCE(
+	NRF_DT_GPIOTE_INST(FEM_NODE, pdn_gpios));
+static uint8_t gpiote_ch_pdn;
+
+static const nrfx_gpiote_t gpiote_csn = NRFX_GPIOTE_INSTANCE(
+	NRF_DT_GPIOTE_INST(DT_BUS(FEM_SPI_DEV_NODE), cs_gpios));
+static uint8_t gpiote_ch_csn;
+#endif
+
+/* These headers require the above gpiote-related variables to be declared. */
+#if defined(PPI_PRESENT)
+#include "radio_nrf5_ppi_gpiote.h"
+#elif defined(DPPI_PRESENT)
+#include "radio_nrf5_dppi_gpiote.h"
+#endif
+
```

Comment:
No, because then the added `gpiote_*` variables no longer could be `static` (that header is included also from other files, so the variables referenced in it would need to be declared) and this I want to avoid.